TimeOfFlight/src/TOFUtils.cc

```diff
@@ -28,42 +29,54 @@ using dd4hep::rec::Vector3D;
 using dd4hep::rec::FixedPadSizeTPCData;
 using CLHEP::RandGauss;
 
-
-dd4hep::rec::Vector3D TOFUtils::getHelixMomAtTrackState(const EVENT::TrackState& ts, double bField){
-    double phi = ts.getPhi();
-    double d0 = ts.getD0();
-    double z0 = ts.getZ0();
-    double omega = ts.getOmega();
-    double tanL = ts.getTanLambda();
-
-    HelixClass helix;
-    helix.Initialize_Canonical(phi, d0, z0, omega, tanL, bField);
-    return helix.getMomentum();
-}
```

Comment:
Does removing this break someone elses code?

Reply:
I don't think so. I think we are safe to remove this.
In the worst case, if somebody have used this specific function from TOFUtils in their local code for some reason, instead of two existing UTIL functions (in LCIO and MarlinUtil), which do the same thing, I think it is a good insentive to switch to UTIL functions. Also, this code is present in dozen of places throughout many packages of MarlinReco, so I would be happy to remove another copy of it, just nicely put in the util function...